pyproject.toml

```
@@ -2,6 +2,74 @@
 build-backend = "hatchling.build"
 requires = ["hatch-vcs", "hatchling"]
 
+[dependency-groups]
+dev = ["rcssmin", {include-group = "doc"}, {include-group = "test"}]
```

what do people think about making `dev` group depend on `test_extra` instead of just `test`?  That way, `dev` will include everything needed to run *all* tests (without `test_extra` some tests will get skipped)

```suggestion
dev = ["rcssmin", {include-group = "doc"}, {include-group = "test_extra"}]
```